sites/partners/page_content/locale_overrides/general.json

```diff
@@ -347,5 +342,6 @@
   "users.resendInvite": "Resend Invite",
   "users.totalUsers": "total users",
   "users.unconfirmed": "Unconfirmed",
-  "users.userDetails": "User Details"
+  "users.userDetails": "User Details",
+  "t.additionalAccessibility": "Additional Accessibility Details"
```

Comment:
We don't need to have this string in both the public strings file and the partners overrides file, because if partners doesn't find a string in this overrides file, it just looks at the one in public - so this one can be removed